modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -2597,10 +2597,13 @@ void GDScriptAnalyzer::reduce_identifier_from_base(GDScriptParser::IdentifierNod
 	const StringName &native = base.native_type;
 
 	if (class_exists(native)) {
-		PropertyInfo prop_info;
 		MethodInfo method_info;
-		if (ClassDB::get_property_info(native, name, &prop_info)) {
-			p_identifier->set_datatype(type_from_property(prop_info));
+		if (ClassDB::has_property(native, name)) {
+			StringName getter_name = ClassDB::get_property_getter(native, name);
+			MethodBind *getter = ClassDB::get_method(native, getter_name);
+			if (getter != nullptr) {
+				p_identifier->set_datatype(type_from_property(getter->get_return_info()));
```

Comment:
`get_return_info()` is only enabled with `DEBUG_METHODS_ENABLED`, so this doesn't compile for all platforms/targets.

Reply:
It just occurred to me that we need to enable some of those things on release builds too. Otherwise typed GDScript won't be working correctly and will miss a bunch of optimizations.